src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/detect_enclosure_freebsd.py

```diff
@@ -0,0 +1,128 @@
+import subprocess as sb
```

Comment:
We dont use this anywhere else in the codebase, I would prefer we keep the same without abbreviations to not confuse other people.